module/os/freebsd/zfs/zvol_os.c

```diff
@@ -1606,8 +1664,7 @@ zvol_busy(void)
 int
 zvol_init(void)
 {
-	zvol_init_impl();
-	return (0);
+	return (zvol_init_impl(mp_ncpus, zvol_num_taskqs, zvol_threads));
```

Comment:
It seems weird to first make variables global and then pass them identically as arguments on both platforms.

Reply:
It is needed to pass ncpu at least, because getting this value is platform-dependent.

Reply:
There is cross-platform `max_ncpus`.